plugins/package-managers/bazel/src/test/kotlin/CompositeBazelModuleRegistryServiceTest.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.bazel
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+
+class CompositeBazelModuleRegistryServiceTest : WordSpec({
+    "CompositeBazelModuleRegistryService.kt.create" should {
```

Comment:
`CompositeBazelModuleRegistryService.kt.create` is misleading as `create()` is not test but
only `URL_REGEX` and `CompositeBazelModuleRegistryService.kt` should be dropped.

I propose to just name it `"URL_REGEX"`.